meerk40t/core/drivers.py

```diff
@@ -292,7 +292,19 @@ def command(self, command, *values):
                     except Exception:
                         pass
                 elif OS_NAME == "Darwin": # Mac
-                    os.system('afplay /System/Library/Sounds/Ping.aiff')
+                    """os.system('afplay /System/Library/Sounds/Ping.aiff')"""
+                    """os.system('plutil -p ~/Library/Preferences/.GlobalPreferences.plist | grep com.apple.sound.beep.sound | cut -d \'"\' -f 4 | xargs afplay')"""
+                    os.system('osascript -e \'display notification "Job Finished" with title "MeerK40t"\'')
```

Comment:
I am positive about this PR - users need something to happen when there is a beep - but request the following changes:

1. This says "Job finished" when the job may not be finished if the Beep is mid-job. I think the notification should be "Meerk40t beep".
2. For people with multiple lasers, it would be useful to know which laser has beeped, and possibly the name of the project that has just finished.
3. The string(s) need to be translatable.

In addition, I have the following more general comments which are for another day / another PR (or this one if optionally you want to add to it):

4. We probably should have a proper "Job Finished" notification function when the spooler goes idle - and it would be great to have this generalised to create Windows notifications too.
5. As an aside, my Octoprint instance for my 3D printer creates pushbullet notifications too. Of course it is far safer to leave a 3D printer unattended than a K40, so I am very unsure whether we should encourage unattended operations by providing remote notifications.

Reply:
P.S. For a generalised Job Finished message, we may need to decide between a generalised popup using wx.adv.NotificationMessage and o/s specific functionality (like this).